ibis/backends/pyspark/tests/test_basic.py

```diff
@@ -214,3 +215,30 @@ def test_alias_after_select(client):
 
     result = table.compile().toPandas()
     tm.assert_series_equal(result['id'], result['id2'], check_names=False)
+
+
+@pytest.mark.parametrize(
+    ('selection_fn', 'expected'),
+    [
+        (lambda t: t[['id']], [True]),
+        (lambda t: t.mutate(v=t['id']), [False, False]),
+        (lambda t: t.mutate(id=t['str_col']), [False, True]),
+        (lambda t: t.mutate(id=t['id']), [True, True]),
+        (lambda t: t.mutate(id=t['id'] + 1), [False, True]),
+        (lambda t: t.relabel({'id': 'id'}), [True, True]),
+        (lambda t: t.relabel({'id': 'id2'}), [False, True]),
+    ],
+)
+def test_can_be_replaced_by_column_name(client, selection_fn, expected):
+    table = client.table('basic_table')
```

Comment:
```
table = ibis.table(('id', datatypes.double), name='test')
```
?

Reply:
This way it's clear which column exists in the table already

Reply:
Updated